### DISCUSSION TITLE
The app does not display any objects in cellular mode.

### DISCUSSION THREAD

**Discussion by laioncenet** [ToolJet/ToolJet#8654] [General]
The app does not display any objects in cellular mode. I would like a solution to this problem.
![erro](https://github.com/ToolJet/ToolJet/assets/154385025/351625b2-9928-43cb-81dc-d2aee8a36671)

**Comment by ShazanRizvi** [comment]
Hey @laioncenet 
Could you share more elaborately about the issue and also could you share the version of ToolJet you are currently using? 

You can also join our slack community to get help from our team and our community members. Here is the invite: https://tooljet.com/slack.

**Comment by laioncenet** [comment]
the platform I'm using is EasyPanel, I'm going to make a video, installing it from scratch and creating the first application in ToolJet, so in cell phone mode it won't display. Only on the computer.

**Comment by laioncenet** [comment]
Solved!
